packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -257,12 +289,119 @@ function WalletProvider({
         [walletState, experimental_getOrCreateWalletWithRecoveryKey]
     );
 
+    // skip wallet creation if using CrossmintAuthProvider
```

Comment:
what do you mean? dont we also want to auto create wallet when using crossmint auth?